src/main/resources/resources.properties

```diff
@@ -37,7 +37,7 @@ notification.CourseFileError.title=Could not access course settings file
 notification.CourseFileError.content=The local file for course settings could not be accessed. Error message: {0}.
 notification.CourseVersionError.title=Plugin incompatible with course
 notification.CourseVersionOutdatedError.content=This course project cannot be loaded because your A+ Courses plugin version is outdated. Please update the A+ Courses plugin to open this course.
-notification.CourseVersionTooNewError.content=This course is no longer supported by your A+ Courses plugin version. Please contact the course staff for assistance.
+notification.CourseVersionTooNewError.content=This course offering requires an older version of the A+ Courses plugin than what you have. Please check that you aren''t accidentally accessing an old course offering from an earlier semester. If you do wish to use the plugin with this course project, reverting to an older version of the plugin may help. If this message does not help you resolve the issue, please contact the course staff for assistance.
```

Comment:
I wasn’t 100% certain what the original text meant, but if it meant what I think it meant, then I’d suggest this replacement for it (even though the new text is very long).